lib/ansible/module_utils/network/meraki/meraki.py

```diff
@@ -195,7 +199,10 @@ def get_nets(self, org_name=None, org_id=None):
             org_id = self.get_org_id(org_name)
         path = self.construct_path('get_all', org_id=org_id, function='network')
         r = self.request(path, method='GET')
-        return r
+        if self.status == 200:
+            return r
+        else:
```

Comment:
So we tend to not do the else if the previous if-block returns from the function. The else is unneeded.